crowdfunding_compassion/controllers/homepage_controller.py

```diff
@@ -6,11 +6,14 @@
 class HomepageController(Controller):
     @route("/homepage", auth="public", website=True)
     def homepage(self, **kwargs):
+        project_obj = request.env['crowdfunding.project']
         context = {
-            "funds": request.env['product.product'].sudo().search([
+            "funds": request.env['product.template'].sudo().search([
```

Comment:
Please keep using product.product

Reply:
But in a past task I had to add to the product.template model the field `activate_for_crowdfunding`. So I thought we should use that

Reply:
What you add in product.template is also added into product.product.

Reply:
These are shared models (inheritance by delegation...)

Reply:
ohhhhh I see now. Ok I will make the changes